src/realm/group.cpp

```diff
@@ -413,16 +413,14 @@ void Transaction::upgrade_file_format(int target_file_format_version)
     // forget it.
     REALM_ASSERT_EX(target_file_format_version == 22, target_file_format_version);
```

Comment:
I agree it is valuable to keep this one hard coded. But I'm less sure about the value of https://github.com/realm/realm-core/blob/bc65972018120af28bb6490b2c18f1454ac97109/test/test_transactions.cpp#L170 inevitably we forget that one because it is only tested as part of the long running tests.

Reply:
I wasn't even aware of this test. Removed the hard coded number. No value in having that.